validators/zone_specific_checks.py

```diff
@@ -1,8 +1,6 @@
-from datetime import datetime
-
 import pandas as pd
 
-from electricitymap.contrib.validators.lib.config import validator
+from validators.lib.config import validator
```

Comment:
I think that these needs to remain as `electricitymap.contrib.validators` for us to be able to import them in the backend. There's a symlink from validators to `electricitymap.contrib.validators`.

Reply:
Ah I was hoping it would work with the direkt link since the symlinks used by poetry doesn't work on windows.

I have changed some of them in the past without issue (I think), see #4256 for more details.

So could you double check if this breaks the backend? Obviously if it does I'll revert it otherwise I'd like to keep the changes.

Reply:
Ah, I see. Let's try to merge this then. I'll create a follow-up PR reverting this if it breaks the tests.

Reply:
Thanks!